x-pack/test/apm_api_integration/common/fixtures/es_archiver/archives_metadata.ts

```diff
@@ -5,10 +5,13 @@
  * 2.0.
  */
 
-/* eslint-disable-next-line*/
-    export default {
+export default {
   'apm_8.0.0': {
     start: '2021-08-03T06:50:15.910Z',
     end: '2021-08-03T07:20:15.910Z',
   },
+  'apm_data_generation_8.0.0': {
```

Comment:
can we find a less generic name than "apm_data_generation".

Perhaps we can rename "apm_8.0.0" to "apm_opbeans_8.0.0", and this archive "apm_synthetic_8.0.0"

Reply:
yes, I can change it, I'll first look into the bug you've mentioned here https://github.com/elastic/kibana/pull/112343#discussion_r719243451